### ISSUE TITLE
Classifier model parameter not trained.

### ISSUE THREAD

**Issue by ksivaman** [IBM/adversarial-robustness-toolbox#142]
I noticed that when using the art pytorch classifier interface and supplying our own model (via parameter model=my_model) , my_model does not really get trained if you save and load the state dict of the model even after applying the '.fit' method. Is there any way to save the trained pytorch model and load it instead of saving the art interface of pytorch classifier?

And if I do save the art interface via 'art.classifiers.pytorch.PyTorchClassifier.save(filepath)', how do I load it back?

**Comment by ksivaman** [comment]
I found how to load it by looking at the source code. It's a bit counter intuitive to save the art classifier as a torch model. I suggest making that a little more clear by including it in the docs maybe.

**Comment by beat-buesser** [comment]
Hi @ksivaman Thank you for the suggestion!

**Comment by SophieChang66** [comment]
@ksivaman hi, I trained one model using classifier.fit afer some epoch, but the accuracy is 10 %. I also found that the pytorch classifier not trained the model's parameter, how do you solve it ?
